packages/components/src/color-picker/stories/index.js

```diff
@@ -49,7 +49,7 @@ const Example = () => {
 		>
 			<ColorPicker { ...props } color={ color } onChange={ setColor } />
 			<div style={ { width: 200, textAlign: 'center' } }>
-				{ colorize( color ).toHslString() }
+				{ colord( color ).toHslString() }
```

Comment:
For the purpose of showing the actual value of the `color` returned in the `onChange` callback, should we change this to just be

```suggestion
				{ color }
```

?